src/jit/importer.cpp

```diff
@@ -3714,6 +3714,34 @@ GenTreePtr Compiler::impIntrinsic(GenTreePtr            newobjThis,
             break;
         }
 
+        case CORINFO_INTRINSIC_GetRawHandle:
+        {
+            assert(IsTargetAbi(CORINFO_CORERT_ABI)); // Only CoreRT supports it.
+            CORINFO_RESOLVED_TOKEN resolvedToken;
+            resolvedToken.hMethod      = method;
+            resolvedToken.tokenContext = MAKE_METHODCONTEXT(info.compMethodHnd);
+            resolvedToken.tokenScope   = info.compScopeHnd;
+            resolvedToken.token        = memberRef;
+            resolvedToken.tokenType    = CORINFO_TOKENKIND_Method;
+
+            CORINFO_GENERICHANDLE_RESULT embedInfo;
+            info.compCompHnd->expandRawHandleIntrinsic(&resolvedToken, &embedInfo);
+
+            GenTreePtr eeTypePtrOfNode = impLookupToTree(&resolvedToken, &embedInfo.lookup,
+                                                         gtTokenToIconFlags(memberRef), embedInfo.compileTimeHandle);
+            if (eeTypePtrOfNode == nullptr)
+                return nullptr;
+
+            unsigned eeSlot = lvaGrabTemp(true DEBUGARG("eeTypePtrOf"));
```

Comment:
Nit: could you remove the references to `EETypePtrOf`?

While the original motivation for this is to allow having a `EETypePtr EETypePtrOf<T>()` method that intrinsically expands to _"raw value of the TypeHandle of T"_, I'm already thinking of using this for `IntPtr DefaultConstructorOf<T>()` that expands to _"pointer to the default constructor of type T"_ (to be used in the implementation of `Activator.CreateInstance<T>`). The API surface is general purpose to allow adding other interesting cases without requiring RyuJIT to learn about them.